sympy/integrals/integrals.py

```diff
@@ -1149,19 +1149,19 @@ def _eval_lseries(self, x, logx, cdir=0):
         for term in expr.function.lseries(symb, logx):
             yield integrate(term, *expr.limits)
 
-    def _eval_nseries(self, x, n, logx, cdir=0):
+    def _eval_nseries(self, x, n, logx=None, cdir=0):
         expr = self.as_dummy()
         symb = x
         for l in expr.limits:
             if x in l[1:]:
                 symb = l[0]
                 break
         terms, order = expr.function.nseries(
-            x=symb, n=n, logx=logx).as_coeff_add(Order)
+            x=symb, n=n).as_coeff_add(Order)
```

Comment:
This change is also not needed, `logx` parameter can be kept, though it won't make a difference